src/components/AppRouter/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { Home } from "../Home"
+import { Login } from "../Login"
+import { BrowserRouter as Router, Route } from "react-router-dom"
+import { Navigation } from "../Navigation"
+
+export const AppRouter = () => {
+  return (
+    <div>
+      <Router>
+        <Navigation />
+        <Route exact path="/login" component={Login} />
+        <Route exact path="/home" component={Home} />
```

Comment:
@Alexandria Do you ever see yourself needing to render the `login` and `home` components at the same time? If not you might want to look into the `Switch` component that `react-router` provides.